libecl/src/ecl_grid.c

```diff
@@ -694,6 +694,7 @@ static void point_copy_values(point_type * p , const point_type * src) {
 #define CELL_FLAG_TAINTED  4     /* lazy fucking stupid reservoir engineers make invalid grid
                                     cells - for kicks??  must try to keep those cells out of
                                     real-world calculations with some hysteric heuristics.*/
+#define CELL_FLAG_VOLUME 8
```

Comment:
> Would it be possible to initialize volume with -1 and skip this flag?

There are cases (not active now but still) - where the signed volume is interesting.